src/Authentication/Program.cs

```diff
@@ -309,7 +309,13 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
     services.Configure<Altinn.Common.AccessToken.Configuration.KeyVaultSettings>(config.GetSection("kvSetting"));
 
     services.Configure<AccessTokenSettings>(config.GetSection("AccessTokenSettings"));
+
+    // Configure OIDC providers used by JwtCookieHandler
```

Comment:
how does this differ from that existing config(the line just below) Rune?

Reply:
Det er forskjelli type slik at JWTCookie Authentication får ikke den andre inn. Den får inn at config null. Se linje 113 og typen som kommer inn. (forskjellige namespace)
https://github.com/Altinn/altinn-authentication/blob/main/src/jwtcookie/Authentication/JwtCookie/JwtCookieHandler.cs#L113